src/extension.ts

```diff
@@ -122,6 +135,80 @@ export function deactivate(): Thenable<void> | undefined {
   return currentClient?.stop();
 }
 
+class KeywordCompletionItemProvider implements CompletionItemProvider {
```

Comment:
We already do it in Metals for Scala 2, we need to also implement it for Scala 3, but let's not do it in the vscode extension.

Reply:
Oks! 
I literally just almost blindly implemented the original repo changes! :D